### PR TITLE
PR 02 - Rspec and testing related gems

### PR DESCRIPTION
**Pull request by ghost**
1. Installed gems for rspec, shoulda matchers, factory bot
2. Installed related gems for testing, such as pry, db cleaner, webdrivers, selenium
3. Initialized rspec and rspec runs with 0 examples.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -8,18 +8,20 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 3.11'
 gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
-
-gem 'coffee-rails', '~> 4.2'
+gem 'haml-rails', '~> 2.0', '>= 2.0.1'
```
Comment (oliverochman) [inline Gemfile]: out of scope

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+# files.
+#
+# Given that it is always loaded, you are encouraged to keep this file as
+# light-weight as possible. Requiring heavyweight dependencies from this file
+# will add to the boot time of your test suite on EVERY test run, even for an
+# individual file that may not need all of that loaded. Instead, consider making
+# a separate helper file that requires the additional dependencies and performs
+# the additional setup, and require it from the spec files that actually need
+# it.
+#
```
Comment (oliverochman) [inline spec/spec_helper.rb]: Clean this file up

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -0,0 +1,100 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```
Comment (oliverochman) [inline spec/spec_helper.rb]: please take a look at how we setup rspec in the course documentation for "Rails AUT". You are missing some configuration

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (ghost) [reply Gemfile]: Put back and will remove later.

File: spec/spec_helper.rb (hunk 2 above)
Comment (ghost) [reply spec/spec_helper.rb]: Done.

File: spec/spec_helper.rb (hunk 3 above)
Comment (ghost) [reply spec/spec_helper.rb]: I think this is fine.  I added the shoulda-matchers to the 'rails-helper', is that what you are referring to?

**Pull request merged**